lib/index.js

```diff
@@ -19,7 +22,8 @@ var defaults = {
   fillGaps: false,
   gradientBackground: false,
   square: false,
-  padding: 0
+  padding: 0,
+  stickerSheet: true
```

Comment:
Default this to `false`.  We don't want any breaking changes in this version of kepler– only new functionality.  This type of defaults change could break the svg server if we didn't account for it.